shotover-proxy/src/frame/cassandra.rs

```diff
@@ -432,3 +481,24 @@ pub enum CassandraResult {
     SchemaChange(SchemaChange),
     Void,
 }
+
+#[derive(PartialEq, Debug, Clone)]
+pub enum BatchStatementTy {
+    Statement(CQL),
+    PreparedId(CBytesShort),
+}
+
+#[derive(PartialEq, Debug, Clone)]
+pub struct BatchStatement {
+    ty: BatchStatementTy,
```

Comment:
Can we just say `type`?

Reply:
nope, `type` is a keyword

Reply:
ahhh I should do BatchStatementTy -> BatchStatementType though